tests/metrics/machine_learning/tensorflow.sh

```diff
@@ -59,7 +64,8 @@ function create_resnet_start_script() {
 
 cat <<EOF >>"${script}"
 #!/bin/bash
-python benchmarks/scripts/tf_cnn_benchmarks/tf_cnn_benchmarks.py -data_format=NHWC --device cpu --batch_size=${BATCH_SIZE} --num_batches=${NUM_BATCHES} > resnet_results
+pushd "benchmarks/scripts/tf_cnn_benchmarks"
+python tf_cnn_benchmarks.py -data_format=NHWC --device cpu --batch_size=${BATCH_SIZE} --num_batches=${NUM_BATCHES} > "/${RESNET_FILE}"
```

Comment:
Is there a reason this command was split into 2 lines?

Reply:
@cmaf thank you, I split the 'launch workload' line just to add clarity to the code